src/containers/App/index.js

```diff
@@ -130,11 +130,16 @@ export default class App extends Component {
 const asyncConnectProps = [{
   promise: ({ store: { dispatch, getState } }) => {
     const promises = [];
+    const state = getState();
 
-    if (!isAuthLoaded(getState())) {
+    if (!isAuthLoaded(state)) {
       promises.push(dispatch(loadAuth()));
     }
 
+    if (state.auth.user) {
+      promises.push(dispatch(loadGroups()));
```

Comment:
Because redux connect is the place where you get what you need for your components.

In this case I move to the top level because header needs it to show groups selector.